src/index.d.ts

```diff
@@ -2,10 +2,11 @@ declare module '*.md';
 
 declare namespace React.JSX {
   interface DivProps extends React.HTMLAttributes<HTMLDivElement> {
-    ref?: ForwardedRef<HTMLDivElement> | null;
+    ref?: ForwardedRef<HTMLElement> | null;
```

Comment:
Revert this to a div
```suggestion
    ref?: ForwardedRef<HTMLDivElement> | null;
```